GIT_AND_GITHUB.md

```diff
@@ -89,6 +77,13 @@ discuss other alternatives. Of course, this means you are putting yourself at ri
 of sharing your knowledge and/or learning some new stuff, so please be careful not
 to end up being even more awesome than you are.
```

Comment:
We could something about what to do as reviewer maybe, where to focus on. Code style, structure, maybe something new you just applied somewhere etc.

Adn point to the guidelines of giving feedack from github: [How to write the perfect PR](https://github.com/blog/1943-how-to-write-the-perfect-pull-request) (and then at the section feedback. 

I like to point out extra that you should always keep in mind your reviewing something someone worked hard for, so always be respectful when giving feedback. Those guidelines from github really help to accomplish this!